builder/ncloud/step_delete_block_storage_instance.go

```diff
@@ -138,10 +138,6 @@ func (s *StepDeleteBlockStorage) deleteVpcBlockStorage(serverInstanceNo string)
 }
 
 func (s *StepDeleteBlockStorage) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
-	if s.Config.BlockStorageSize == 0 {
```

Comment:
I understand the situation better now thank you. So if a server has block storage then when Packer creates a new server from the Image the running instance will have attached block storage as well, which is what causes the issue. Because a server can not be terminated until all additional storage has been deleted. 

Since this only an issue when using an existing server image (s.Confg.MemberServerImageNo != "") why not add a check for it in the conditional?

```go
if s.Config.BlockStorageSize == 0 && s.Config.MemberServerImageNo != "" {
return processStepResult(nil, s.Error, state)
}
```

This will ensure that anytime an existing server image is used a call to delete will execute. Otherwise it will only execute when additional storage has been added. 

The change as you have will always execute a Delete request, which doesn't seem necessary.

Reply:
When using an existing server image (=member_server_image_no setting), packer need to run a delete.

So, change to
```
&& s.Config.MemberServerImageNo == ""
```

Reply:
Ah that makes sense. Something made me pause when I commented originally but couldn't put my finger on it. You are right it should be `s.Config.MemberServerImageNo == ""`.